server/analytics/processed_items_report.py

```diff
@@ -0,0 +1,104 @@
+from superdesk import get_resource_service
+from superdesk.services import BaseService
+
+from superdesk.metadata.item import metadata_schema
+from superdesk.resource import Resource
+
+
+class ProcessedItemsResource(Resource):
+    """Processed items schema
+    """
+
+    schema = {
+        'user': metadata_schema['original_creator'],
+        'starting_time': {'type': 'datetime'},
+        'ending_time': {'type': 'datetime'},
+        'report': {'type': 'dict'}
+    }
+
+    item_methods = ['GET', 'DELETE']
+    resource_methods = ['POST']
+
+    privileges = {'POST': 'processed_items_report', 'DELETE': 'processed_items_report', 'GET': 'processed_items_report'}
+
+
+class ProcessedItemsService(BaseService):
+
+    def create_query(self, report):
+        """
+        Returns a query on users.
+        """
+        archive_versions_query = {
+            "task.user": str(report['user'])
+        }
+        return archive_versions_query
+
+    def get_items(self, query):
+        """
+        Return the result of the item search by the given query.
+        """
+        return get_resource_service('archive_versions').get(req=None, lookup=query)
+
+    def count_items(self, items_list, state, starting, ending):
+        """
+        Return the number of items which were modified in a given time interval and having a certain state.
+
+        If no state is give, it will return all items modified in the given time interval.
+
+        :param items_list: list of items based on the query
+        :param state: item's state
+        :param starting: starting time of the given interval
+        :param ending: ending time of the given interval
+        """
+        if state == '':
+            items = sum(1 for i in items_list
+                        if (i['state'] in ['published', 'spiked', 'corrected', 'killed']) and
+                        str(i['_updated']) >= str(starting) and str(i['_updated']) <= str(ending))
```

Comment:
This is not a safe way to compare dates because you don't know the formatting. Please use datetime type.